projects/premake5.lua

```diff
@@ -55,7 +56,147 @@ CreateWorkspace({name = "voxelate"})
 	project("fastlz")
 		language("C")
 		kind("StaticLib")
+		warnings "Off"
 		files({
 			"../fastlz/fastlz.c",
 			"../fastlz/fastlz.h",
 		})
+
+    project("reactphysics3d")
+        language "C++11"
+        kind "StaticLib"
+		symbols "On"
+		warnings "Off"
+
+        includedirs {
+            "../reactphysics3d/src/",
+        }
+
+        files {
+            "../reactphysics3d/src/configuration.h",
```

Comment:
The benefit would be if they add some new files, we can just update the submodule instead of doing that as well as updating the premake files